drivers/st7735/include/st7735_params.h

```diff
@@ -104,7 +107,7 @@ extern "C" {
 /**
  * @brief   Configure LCD
  */
-static const lcd_params_t st7735_params[] =
+static lcd_params_t st7735_params[] =
```

Comment:
Here and in a couple of other drivers. It's a bad idea to remove the `const` keyword in parameter set declarations because it prevents that they are placed in ROM.

The correct approach would be to consider the `spi_freq` in the parameter set as requested frequency and to store the actual frequency in the device descriptor which is then used in `spi_acquire`. This approach would require only 4 bytes of RAM instead of 20 byte as in this driver.

Reply:
Fine, I'll correct it